README.md

```diff
@@ -1,3 +1,39 @@
 # RHMAP Nagios for Docker
 
+[![Build Status](https://travis-ci.org/feedhenry/nagios-container.svg?branch=master)](https://travis-ci.org/feedhenry/nagios-container)
+
 Nagios server in a Docker container for RHMAP.
+
+## Development
+
+Install dependencies:
+```
+pip install pep8
+pip install autopep8
+```
+
+Run tests:
+
+```
+python -m unittest discover -s plugins/default
+````
+
+Style Guide:
+
+All python modules should conform to the PEP8 style guide https://www.python.org/dev/peps/pep-0008/
+
+To verify:
+
+```
+pep8 plugins/ --show-source
```

Comment:
Updated to make the command the same. The  E501 error is about line lengths, you can make it allow longer lines but I think its safe enough to ignore them all for now.